src/main/java/nextstep/courses/domain/sessionuser/SessionUsers.java

```diff
@@ -18,16 +17,24 @@ public SessionUsers(List<SessionUser> sessionUsers) {
         this.sessionUsers = sessionUsers;
     }
 
-    public SessionUser addUser(NsUser nsUser, Session session) {
-        if (!session.canRegisterNewUser(sessionUsers.size())) {
-            throw new ExceedMaxAttendanceCountException("이미 최대 수강 인원이 다 찼습니다.");
+    public long notCanceledUserSize() {
+        return sessionUsers.stream().filter(el -> !el.isCanceled())
+                .count();
+    }
+
+    public void add(SessionUser sessionUser) {
+        Optional<SessionUser> savedSessionUser = sessionUsers.stream().filter(el -> el.getUserId() == sessionUser.getUserId())
```

Comment:
```suggestion
        Optional<SessionUser> savedSessionUser = sessionUsers.stream()
                .filter(el -> Objects.equals(el.getUserId(), sessionUser.getUserId()))
                .findFirst();
```

저는 디버깅과 가독성을 위해서 한줄에 점 하나씩 표현하는 편입니다 ㅎ